From: Hiram Chirino <chirino@gmail.com>
Date: Fri, 5 Jul 2013 18:32:28 +0400
Subject: [PATCH] Allow leveldbjni build


diff --git a/include/leveldb/slice.h b/include/leveldb/slice.h
index a86e8a6..a817395 100644
--- a/include/leveldb/slice.h
+++ b/include/leveldb/slice.h
@@ -82,7 +82,6 @@ class LEVELDB_EXPORT Slice {
             (memcmp(data_, x.data_, x.size_) == 0));
   }
 
- private:
   const char* data_;
   size_t size_;
 };
